### PR TITLE
modernize build

### PR DESCRIPTION
**Pull request by isidroas**
Following recent recomendations of setuptools: https://setuptools.pypa.io/en/latest/userguide/quickstart.html

Also, in my system, the script was not generated in the intallation.

### CODE REVIEW

**Comment by elmotec** [comment]
Thanks @isidroas.  That looks good, I feel we need to update the the appveyor.yml [here](https://github.com/elmotec/massedit/blob/0229b40361ed6e752beb81a69f9f78d92c4906d5/.appveyor.yml#L10) and [here](https://github.com/elmotec/massedit/blob/0229b40361ed6e752beb81a69f9f78d92c4906d5/.appveyor.yml#L15) for the PR to pass.  I feel pip is the right choice now so something like `%python% -m pip install .` should do.  Ok to add that to the PR?

**Comment by isidroas** [comment]
Done, I opted for the first option because it is more minimalist

**Comment by elmotec** [comment]
Thanks!

**Comment by elmotec** [comment]
@isidroas, one more thing: it looks like appveyor does not support old version of Python.  I could update the main branch to use python 3.8 and above only per https://devguide.python.org/versions/ but I feel it will trigger conflicts with your branch.  Could you change [.appveyor.yml](https://github.com/elmotec/massedit/blob/0229b40361ed6e752beb81a69f9f78d92c4906d5/.appveyor.yml#L7) to remove Python27, Python34 so the PR passes?

File: pyproject.toml [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
+     "mass"
 ]
-build-backend = "setuptools.build_meta"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Environment :: Console",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3.4",
```
Comment (elmotec) [inline pyproject.toml:22]: Also remove line 21 and 22 here.  Those versions are no longer supported.  I'll put a separate PR to add 3.10, 3.11. Man, time flies.

**Review by elmotec — CHANGES_REQUESTED** [review]
As mentioned [above](https://github.com/elmotec/massedit/pull/16#issuecomment-1712923429), could you remove the references to deprecated versions of Python so the tests pass?

File: pyproject.toml (hunk 1 above)
Comment (elmotec) [reply pyproject.toml:22]: And in [.github/workflows/python-app.py](https://github.com/elmotec/massedit/blob/0229b40361ed6e752beb81a69f9f78d92c4906d5/.github/workflows/python-app.yml#L18) as well.

**Comment by isidroas** [comment]
Changed also _.github/workflows/python-app.py_. I wonder why it is not running, only the appveyor :thinking:

**Comment by elmotec** [comment]
> Changed also _.github/workflows/python-app.py_. I wonder why it is not running, only the appveyor 🤔

Yeah. Not sure. I'll check that out separately.  Thanks for the PR.

**Pull request merged**